pulp_rpm/app/models/package.py

```diff
@@ -283,7 +286,17 @@ def nevra_short(self):
 
     class Meta:
         default_related_name = "%(app_label)s_%(model_name)s"
-        unique_together = ("name", "epoch", "version", "release", "arch", "checksum_type", "pkgId")
+        unique_together = ("_pulp_domain", "pkgId")
+        unique_together = (
```

Comment:
Does this actually work? I think it should be:
```suggestion
        unique_together = (("_pulp_domain", "pkgId"), ("_pulp_domain", "name", ...))
```

Reply:
Yes, needs to be a tuple of tuples.

Reply:
And i think the first one is interpreted as the `natural_key`.

Reply:
As it turns out, There Can (and should) Be Only One, and the second (correct) one is the one chosen. Fixed.